torch/nn/modules/container.py

```diff
@@ -419,12 +417,12 @@ def update(self, modules: Mapping[str, Module]) -> None:
 class ParameterList(Module):
     r"""Holds parameters in a list.
 
-    :class:`~torch.nn.ParameterList` can be indexed like a regular Python
-    list, but parameters it contains are properly registered, and will be
-    visible by all :class:`~torch.nn.Module` methods.
+    :class:`~torch.nn.ParameterList` can be used like a regular Python
+    list, but Tensors that are :class:`~torch.nn.Parameter` are properly registered,
+    and will be visible by all :class:`~torch.nn.Module` methods.
```

Comment:
doc nit: can we have a comment here explaining that if a tensor is appended to the list, it will be converted to a parameter?

Edit: Ah, I see the note was added below for `ParameterDict` - would be good to replicate it here as well